src/main/java/io/jenkins/plugins/customizable_header/UserHeader.java

```diff
@@ -123,15 +129,27 @@ public String getDisplayName() {
       return "Customizable Header";
     }
 
-    //        @Override
-    //        public @NonNull UserPropertyCategory getUserPropertyCategory() {
-    //            return UserPropertyCategory.get(UserPropertyCategory.Appearance.class);
-    //        }
-
-    // replace with above method when bumping core to version including:
-    // https://github.com/jenkinsci/jenkins/pull/7268
-    public @CheckForNull String getUserPropertyCategoryAsString() {
-        return "appearance";
+    @Override
+    public @NonNull UserPropertyCategory getUserPropertyCategory() {
+        return UserPropertyCategory.get(UserPropertyCategory.Appearance.class);
     }
   }
+
+  // TODO - this should be combined with CustomHeaderConfiguration#401
+  public List<Descriptor<Logo>> getLogoDescriptors() {
+    return LogoDescriptor.all().stream()
+            .filter(d -> !(d instanceof Icon.DescriptorImpl))
+            .collect(Collectors.toList());
+  }
+
+  // TODO - this should be combined with CustomHeaderConfiguration#407
```

Comment:
TODO

Reply:
Moved that part to the HeaderColor class